### DISCUSSION TITLE
PPOCR Lable not working

### DISCUSSION THREAD

**Discussion by riyajatar93** [PaddlePaddle/PaddleOCR#12714] [Q&A]
PPOCRLabel 
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
QObject::moveToThread: Current thread (0x20743f0) is not the object's thread (0x5e08660).
Cannot move to target thread (0x20743f0)

qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/home/riyaj-atar/anaconda3/envs/labeling/lib/python3.8/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb, eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl.

--------------------------------------
C++ Traceback (most recent call last):
--------------------------------------
No stack trace in paddle, may be caused by external reasons.

----------------------
Error Message Summary:
----------------------
FatalError: `Process abort signal` is detected by the operating system.
  [TimeInfo: *** Aborted at 1696489154 (unix time) try "date -d @1696489154" if you are using GNU date ***]
  [SignalInfo: *** SIGABRT (@0x3e8000404cf) received by PID 263375 (TID 0x7fa90b64a440) from PID 263375 ***]

Aborted (core dumped)

**Comment by a871545252** [comment]
I meet the same problem.

**Comment by a871545252** [comment]
Did you resolve this problem?

**Comment by HuyTT1611** [comment]
```python
pip install opencv-python-headless==4.5.3.56
```

**Comment by hopegithub** [comment]
pip install paddlepadle==2.5.2 和pip install opencv-python-headless==4.5.3.56

**Comment by SWHL** [comment]
Since PPOCRLabel has moved, please go to [PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel) to ask questions if you have any questions!
This issue has been closed first.
